xrpl/asyncio/ledger/main.py

```diff
@@ -42,9 +49,21 @@ async def get_latest_open_ledger_sequence(client: Client) -> int:
     Raises:
         XRPLRequestFailureException: if the rippled API call fails.
     """
-    response = await client._request_impl(Ledger(ledger_index="open"))
+    response = await client._request_impl(Ledger(ledger_index="current"))
+    assert (
+        response.result["ledger"]["closed"] is False
+    ), "`closed` key is not relevant in current ledger"
+    assert (
+        response.result["validated"] is False
+    ), "`validated` key is not relevant in current ledger"
+    assert (
+        "ledger_index" not in response.result
+    ), "`ledger_index` key is not relevant in current ledger"
+    assert (
+        "ledger_current_index" in response.result
+    ), "`ledger_current_index` key must be present in current ledger"
```

Comment:
Why are you using assert statements in non-test modules?

Reply:
thanks. I have updated them to throw Exceptions now. 25e7304c